### PR TITLE
Closes #3: Update README files

### LINKED ISSUE #3: Update README files
Update README files after restructuring the project and moving it from SVN.

### PR DESCRIPTION
**Pull request by mkobos**
- I added main README file with embedded diagram of the OpenAIRE system.
- I changed location of some of the READMEs to make them fit the new structure of the code base.
- I updated the content of the READMEs.
- Other minor clean-up in the documentation.

### CODE REVIEW

**Comment by marekhorst** [comment]
Looks pretty good to me. Let's merge it.

**Comment by mkobos** [comment]
C'mon, maybe at least one comment on what I should modify - preferably next to the line that I should change? I'd love to check how uploading a new version of the code will work here.

File: iis-assembly-resources/README.markdown [hunk 1]
```diff
@@ -1,5 +1,7 @@
-Module utilized by `icm-iis-parent-container`.
+Module utilized by `iis-parent-container`.
```
Comment (marekhorst) [inline iis-assembly-resources/README.markdown]: In fact it is now called 'iis-workflows' parent module.

File: iis-properties-maven-plugin/README.markdown [hunk 2]
```diff
@@ -1,6 +1,6 @@
-Maven plugin module utilized by `icm-iis-parent-container` for proper `job.properties` file building.
-Based on:
-http://site.kuali.org/maven/plugins/properties-maven-plugin/1.3.2/write-project-properties-mojo.html
-Supplemented with:
+Maven plugin module utilized by `iis-parent-container` for proper `job.properties` file building.
```
Comment (marekhorst) [inline iis-properties-maven-plugin/README.markdown]: In fact it is now called 'iis-workflows' parent module.

File: iis-assembly-resources/README.markdown (hunk 1 above)
Comment (mkobos) [inline iis-assembly-resources/README.markdown]: Ok. I'll also search through the code for other occurrences of `iis-parent-container` and change them to `iis-workflows`.

**Comment by mkobos** [comment]
Please take a look at the changes addressing your comments that I introduced.

**Comment by marekhorst** [comment]
Seems to be fine.

**Pull request closed without merging**